src/index.html

```diff
@@ -11,7 +11,174 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application">
+
+      <fieldset class="fieldset">
+        <legend>Personal Information</legend>
+
+        <label class="person-info">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            required
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="person-info">
+          Name:
+          <input
+            type="text"
+            name="name"
+            required
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="person-info">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            min="1"
+            max="100"
+            value="12"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="person-info">
+          Full date of birth:
+          <input
+            type="date"
+            name="bithday"
+          >
+        </label>
+
+        <label class="person-info">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="accept"
+          >
+        </label>
+
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Regestration</legend>
+
+        <label class="person-info">
+         E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+          >
+        </label>
+
+        <label class="person-info">
+          Password
+          <input
+            type="password"
+            name="password"
+            minlength="5"
+            maxlength="32"
+          >
+        </label>
+
+      </fieldset>
+
+
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+
+        <label class="person-info">
+          Do you love cats?
```

Comment:
Clicking on the `Do you love cats?` text will automatically select the `Yes` option - wrong behavior
```suggestion
        <div class="person-info">
          Do you love cats?
```